TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/Compiler.php

```diff
@@ -141,6 +141,9 @@ public function getProxyClass($fullClassName)
         if (in_array(substr($fullClassName, 0, 14), $this->blacklistedSubPackages)) {
             return false;
         }
+        if ($classReflection->isFinal() && preg_match('/^\s?\*\s?\@Annotation\s/m', $classReflection->getDocComment()) === 1) {
```

Comment:
What is the purpose of this? I understand it skips classes that have `@Annotation` in their docblock, but why?